source/public/styles/styles2.css

```diff
@@ -0,0 +1,415 @@
+    
+}
+
+#projects{
+    display: flex;
+    flex-direction: column;
+    border: 1px solid white;
+    width: 25%;
+    background-color: white;
+    gap:20px;
+    border-radius: 10px;
+    
+}
+
+.project-item{
+    width: 80%;
+    margin: 0 auto;
+
+}
+
+.prioritytype{
+    width:75%;
+    background-color:white;
+    border-radius: 32px;
+    color:#FF5353;
+
+}
+
+.edit-project{
+    background-color: transparent;
+    border:none;
+
+
+}
+
+.project-item-delete{
+    
+    background-color: transparent;
+    border:none;
+}
+.fa-trash{
+    color:red;
+    font-size:1.25rem;
+   
+}
+
+.fa-pencil{
+    font-size:1.25rem;
+}
+
+#project-modal{
+   display:none; /*comment this to see modal*/
+   /*display:flex;*/ /*uncomment to see modal*/
+   position:fixed;
+   top:0;
+   left:0;
+   width:100%;
+   height:100%;
+   background-color: rgba(0,0,0,0.7);
+   z-index: 1000;
+   justify-content: center;
+   align-items: center;
+   
+}
+
+.modal-content{
+    display: flex;
+   background-color: #fff;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+    max-width: 500px;
+    width: 90%;
+    aspect-ratio: 16/9;
+    align-items: center;
+}
+
+#project-name{
+    height:100%;
+    border-radius: 10px;
+}
+
+#project-form{
+    height:90%;
+    margin: auto;
+    width:100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    .input-group{
+        display:flex;
+        flex-direction: column;
+        width:80%;
+        margin: 0 auto;
+        gap:6px;
+        height:33%;
+        border-radius: 10px;
+    }
+}
+
+
+
+#tag-select{
+    display:flex;
+    width:80%;
+    margin:0 auto;
+    gap: 15%;
+    .input-group{
+        display:flex;
+        flex-direction: column;
+        width:80%;
+        margin: 0 auto;
+        gap:6px;
+        height:100%;
+        border-radius: 10px;
+    }
+    height: 33%;
+    border-radius: 10px;
+}
+
+.button-group{
+    width:80%;
+    margin:0 auto;
+    display:flex;
+    justify-content: space-between;
+
+    
+}
+
+.sub-button-group{
+    display:flex;
+    gap:20px;
+
+}
+
+.save{
+    background-color: #008000;
+    color:white;
+    border-radius: 10px;
+    padding:8px 16px;
+    border:none;
+}
+
+.cancel{
+    background-color: #808080;
+    color:white;
+    border-radius: 10px;
+    padding:8px 16px;
+    border:none;
+
+}
+.delete{
+    background-color: #FF0000;
+    color:white;
+    border-radius: 10px;
+    padding:8px 16px;
+    border:none;
+}
+
+#delete-project{
+    display:flex;
+    justify-content: flex-end;
+}
+
+#deadline{
+    height:100%;
+    border-radius: 10px;
+    padding:10%;
+}
+
+.newProjectButton{
+    display: flex;
+    justify-content: center;
+    font-size: medium;
+    width: 80%;
+    margin: 0 auto;
+    padding: 10px;
+    align-items: center;
+    color: white;
+    background-color: #976393;
+    border-radius: 5px;
+    border:none;
+}
+
+h2{
```

Comment:
Just a heads up to watch out for repeated query selectors when combining the stylesheets if the rules override each other